packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -159,6 +159,23 @@ class MediaQueryData {
   /// pixels are the size of the actual hardware pixels on the device. The
   /// number of physical pixels per logical pixel is described by the
   /// [devicePixelRatio].
+  ///
+  /// ## Troubleshooting
+  ///
+  /// Beware that it is considered bad practice to cache and later use the size
```

Comment:
```suggestion
  /// It is considered bad practice to cache and later use the size
```
"Beware that" is basically the same as "Note:", which we discourage in the docs.

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-empty-prose